### PR TITLE
refactor: Chaos structure refactor

### PR DESCRIPTION
**Pull request by GuyAb**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔✖                                                                        |
| New feature     | ✖                                                                      |
| Breaking change |✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |

Related issues: #XXX , #XXX ...
Closes #XXX , #XXX ...

Further  information:

### CODE REVIEW

File: src/chaos-experiments/models/chaosExperimentsManager.js [hunk 1]
```diff
@@ -92,17 +93,49 @@ module.exports.insertChaosJobExperiment = async (jobExperimentId, jobId, experim
 
 module.exports.runChaosExperiment = async (kubernetesChaosConfig, jobExperimentId) => {
     try {
-        await kubernetesConnector.runChaosExperiment(kubernetesChaosConfig);
+        console.log('connector: ' + connector);
```
Comment (kerenfi) [inline src/chaos-experiments/models/chaosExperimentsManager.js:96]: remove log

File: tests/unit-tests/k8sChaosExperiments/models/chaosExperimentsManager-test.js [hunk 2]
```diff
@@ -31,12 +32,14 @@ describe('Chaos experiments manager tests', function () {
...
         runChaosExperimentConnectorStub = sandbox.stub(chaosExperimentConnector, 'runChaosExperiment');
+        getFutureJobExperimentsStub = sandbox.stub(database, 'getFutureJobExperiments');
+        manager.__set__('connector', require('../../../../src/chaos-experiments/models/kubernetes/chaosExperimentConnector'));
     });
     after(() => {
         sandbox.restore();
     });
 
-    beforeEach(() => {
+    beforeEach(async() => {
```
Comment (kerenfi) [inline tests/unit-tests/k8sChaosExperiments/models/chaosExperimentsManager-test.js:42]: remove async

Comment (GuyAb) [reply tests/unit-tests/k8sChaosExperiments/models/chaosExperimentsManager-test.js:43]: now I need it, take a look again

File: src/chaos-experiments/models/chaosExperimentsManager.js (hunk 1 above)
Comment (GuyAb) [reply src/chaos-experiments/models/chaosExperimentsManager.js:94]: done

**Pull request merged**